firetv/__init__.py

```diff
@@ -116,57 +116,215 @@ def connect(self):
                 signer = Signer(self.adbkey)
 
                 # Connect to the device
-                self._adb = adb_commands.AdbCommands().ConnectDevice(serial=self.host, rsa_keys=[signer])
+                self.adb = adb_commands.AdbCommands().ConnectDevice(serial=self.host, rsa_keys=[signer])
             else:
-                self._adb = adb_commands.AdbCommands().ConnectDevice(serial=self.host)
+                self.adb = adb_commands.AdbCommands().ConnectDevice(serial=self.host)
```

Comment:
Why does the `adb` attribute need to be public?

Reply:
In the HA component, I check the `adb` attribute in a few places (e.g., https://github.com/JeffLIrion/home-assistant/blob/dev-firetv-adb/homeassistant/components/media_player/firetv.py#L63). The reviewers didn't like that I was accessing a hidden attribute.

Reply:
https://github.com/JeffLIrion/home-assistant/blob/dev-firetv-adb/homeassistant/components/media_player/firetv.py#L101

```python
        except self.exceptions:
            _LOGGER.error('Failed to execute an ADB command; will attempt to '
                          're-establish the ADB connection in the next update')
            returns = None
            self.firetv.adb = None
```

Here, I set the `adb` attribute to `None` in the event of an ADB error, which will trigger a reconnect attempt in the next update.

Reply:
I changed it back to being hidden (`_adb`) in https://github.com/happyleavesaoc/python-firetv/pull/65/commits/01b985386a4b2c4551cdd0c6e11f30f10f84cf59.